config/sidekiq.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
I guess it doesn't matter. I was thinking just something saying that, by default, DJ is used and this file doesn't matter.